README.md

```diff
@@ -1,17 +1,123 @@
 
-### I'm working on a new channel
-Please create an issue for it if it does not already exist, then PR you code for review.
+This package makes it easy to send notifications using [AllMySms](https://www.allmysms.com/) with Laravel 5.5+ and 6.0
 
-## Workflow for new channels
+## Contents
 
-1) Head over to the [skeleton repo](https://github.com/laravel-notification-channels/skeleton) & press "Use this template" to create a repo from the skeleton.
-2) Use find/replace to replace all of the placeholders with the correct values (package name, author name, email, etc).
-3) Implement to logic for the channel & add tests.
-4) Fork this repo, add it as a remote and push your new channel to a branch.
-5) Submit a new PR against this repo for review.
+- [Installation](#installation)
+	- [Setting up the AllMySms service](#setting-up-the-AllMySms-service)
+- [Usage](#usage)
+	- [Available Message methods](#available-message-methods)
+- [Changelog](#changelog)
+- [Testing](#testing)
+- [Security](#security)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
 
-Take a look at our [FAQ](http://laravel-notification-channels.com/) to see our small list of rules, to provide top-notch notification channels.
+
+## Installation
+
+You can install the package via composer:
+
+``` bash
+composer require laravel-notification-channels/all-my-sms
+```
+
+### Setting up the AllMySms service
+
+Add the following code to you `config/services.php`:
+
+```php
+// config/services.php
+...
+'all_my_sms' => [
+    'uri' => env('ALL_MY_SMS_URI', 'https://api.allmysms.com/http/9.0'),
+    'login' => env('ALL_MY_SMS_LOGIN'),
+    'api_key' => env('ALL_MY_SMS_API_KEY'),
+    'format' => env('ALL_MY_SMS_FORMAT', 'json'),
+    'sender' => env('ALL_MY_SMS_SENDER'),
+    'universal_to' => env('ALL_MY_SMS_UNIVERSAL_TO'),
+],
+...
+```
+
+## Usage
+
+Now you can use the channel in your `via()` method inside the notification:
+
+``` php
+use NotificationChannels\AllMySms\AllMySmsChannel;
+use NotificationChannels\AllMySms\AllMySmsMessage;
+use Illuminate\Notifications\Notification;
+
+class ProjectCreated extends Notification
+{
+    public function via($notifiable)
+    {
+        return [AllMySmsChannel::class]; // or 'all_my_sms'
+    }
+
+    public function toAllMySms($notifiable)
+    {
+        return new AllMySmsMessage('Content');
+    }
+}
+```
+
+In order to let your Notification know which phone number to use, add the `routeNotificationForAllMySms` method to your Notifiable model.
+
+This method needs to return a phone number.
+
+```php
+public function routeNotificationForAllMySms(Notification $notification)
+{
+    return $this->phone_number;
+}
+```
+
+For testing purpose you may set the `ALL_MY_SMS_UNIVERSAL_TO` environment variable to always send SMS to the configured number.
```

Comment:
I'd prefer if this was renamed to indicate it was for testing :)

Reply:
Maybe I should mention `Local development` like the Laravel mail documentation : https://laravel.com/docs/6.x/mail#mail-and-local-development